### PR TITLE
F3-01 -  ingredients warning

### PR DESCRIPTION
**Pull request by CameronVandenBerg17**
The added functionality in ingredient.html will highlight the newly-created <span>s with either red or green depending on the value given to them by the respective variable at the end. Exact style is subject to change and is mostly for placeholder, but functionality is there.

### CODE REVIEW

File: example/flaskr/ingredients.sql [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
```
Comment (scheffeltravis) [inline example/flaskr/ingredients.sql]: I think you might have inadvertently changed the name of this file in your branch when you pulled it down as this file already exists in the repo as `/flaskr/ingredient.sql` (without the 's').
I think we should be good to remove this.

File: example/flaskr/templates/recipe/ingredient.html [hunk 2]
```diff
@@ -15,16 +25,38 @@ <h1>{{ ingredient['name'] }}</h1>
...
+                    //If (the number at the end of the string) is greater or less than value, highlights.
+                    if (Number(valueString.innerHTML.substr(valueString.innerHTML.length - 3, valueString.innerHTML.length)) <= 2.5) {
+                        valueString.classList.add("badingredient");
+                    } else if (Number(valueString.innerHTML.substr(valueString.innerHTML.length - 3, valueString.innerHTML.length)) >= 4.0) {
+                        valueString.classList.add("goodingredient");
+                    }
+                }
+
+                //debug
+                //alert(Number(nourishment.substr(nourishment.length - 3, nourishment.length)
```
Comment (scheffeltravis) [inline example/flaskr/templates/recipe/ingredient.html]: Could probably get rid of this debugging code if it's not needed anymore.

**Comment by scheffeltravis** [comment]
This JS looks pretty good and I don't see any problems with it, but having screenshots in future PRs are helpful for others to see what it is you're trying to accomplish. Great job though man! 👍

File: example/flaskr/ingredients.sql (hunk 1 above)
Comment (CameronVandenBerg17) [reply example/flaskr/ingredients.sql]: I don't know how that happened. I never went and edited file names. Weird, but fixed

**Review by scheffeltravis — APPROVED** [review]

**Comment by elaguerta** [comment]
Found a couple of bugs in the embedded js. I committed my fixes, and now `ingredient/milk-generic` looks like this:
image

**Review by elaguerta — APPROVED** [review]
Approved with a couple commits for bug fixes. Great job!

**Pull request merged**